### PR TITLE
Cart management

### PR DESCRIPTION
**Pull request by Balakumar3018**
Below functionalities are implemented  :- 
1. "Add to cart" from product page
2. After adding "add to cart" it changes to "go to cart "
3. Remove from cart in cart page
4. Increase and decrease quantity in cart page
5. Price card updating while increasing and decreasing item count

- " Yet to add MOVE to WishList functionality "
-
preview-  https://deploy-preview-3--roaring-belekoy-54232b.netlify.app

Yet to add responsiveness

### CODE REVIEW

File: src/components/CartList/CartList.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.cart-page-list{
+    padding-top: 16px;
+}
+.horizontal-card{
+    background-color: var(--background-colorWhite);
+    width: fit-content;
+    box-shadow: 0px 4px 8px var(--secondary-color);
+}
+.card-image{
+    width: 250px;
```
Comment (sumitch29) [inline src/components/CartList/CartList.css:10]: use rem

**Review by sumitch29 — APPROVED** [review]
Home screen shop now button not working below iphone-13 now in green.
Functionality working fine.
suggestion: CSS of cart icon on product page, number is not visible properly.

**Comment by Balakumar3018** [comment]
thanks sumit. will add the suggestions

**Review by dusariudaykumar — APPROVED** [review]
The code is clean and your file structure is also good.
Everything is well structured.

**Pull request merged**